public/js/velauctor.js

```diff
@@ -1,49 +1,63 @@
-var signInOuts = document.querySelectorAll('.dom-sign-in'),
```

Comment:
Most changes here were commenting sections of code and adding some event handlers for the main page.